src/Icons/src/IconRenderer.php

```diff
@@ -16,7 +16,7 @@
  *
  * @internal
  */
-final class IconRenderer
+final class IconRenderer implements IconRendererInterface
```

Comment:
```suggestion
final class RegistryIconRenderer implements IconRendererInterface
```

This could be done in the follow-up PR for #1767.

Reply:
So i revert the change, or i commit your suggestion ?

Reply:
We can leave as is